docs/ref/contrib/syndication.txt

```diff
@@ -1095,3 +1112,47 @@ For example, you might start implementing an iTunes RSS feed generator like so::
 
 There's a lot more work to be done for a complete custom feed class, but the
 above example should demonstrate the basic idea.
+
+Feed stylesheets
+----------------
+
+You can add styling to your RSS feed using an `XSLT stylesheet`_ by passing
+an ``xsl_stylesheet_url`` argument to the feed's constructor.
```

Comment:
Just highlighting this, it may well be fine -- `CTRL-F` suggests this is the only place in the document where we refer to the `__init__` as a _constructor_, I can't tell at a glance what the rest of the document uses as nomenclature 🤷

Reply:
Thanks for pointing this out, I had actually gotten myself mixed up between `django.contrib.syndication.views.Feed` and `django.utils.feedgenerator.SyndicationFeed` and the documentation I'd written didn't actually work.

`SyndicationFeed` is the one that takes arguments in its constructor (this behavior worked and was tested) but `Feed` is customized by settings arguments on the class (this behavior had not been tested and did not work).

I've fixed the missing code and updated the docs 👍🏻